README.md

```diff
@@ -224,16 +226,16 @@ huggingface-login
 
 # Build TRT LLM engine and generate a Triton model repository pointing at it
 triton remove -m all
-triton import -m llama-3-8b-instruct --backend tensorrtllm
+triton import -m llama-3-8b-instruct --backend trtllm
```

Comment:
Has this not been updated in genai-perf yet? I thought genai-perf was already updated to use `tensorrtllm` instead

Reply:
Oh I see in your other comments it's coming in 24.05, hmm. I am against changing this flag back and forth or being consistent for the next few weeks.

Can the special arg parse handling you're already doing for the `profile` subcommand just add a special exception to map `--backend tensorrtllm` to `--backend trtllm` until fixed in later genai-perf version?

Don't want to change the front-facing args on Triton CLI

Reply:
It will be updated in 24.05, but is not updated in 24.04. I updated the logic to do a direct substitution of tensorrtllm to trtllm when passed to GenAI-Perf but missed updating it here. I did so now. I have an open ticket to fix that substitution hotfix once 24.05 is released and used in the CLI.

Reply:
Sorry, my page didn't upload to reflect your latest comment. Yes, that's exactly what's being done. :) Just didn't reflect it here.

I dropped the special arg handling though, since we're not going to be handling that case anymore.